src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -690,6 +686,9 @@ private function getFiltersParameters(string $resourceClass, string $operationNa
             }
 
             foreach ($filter->getDescription($resourceClass) as $name => $data) {
+                if (!preg_match('/^[a-zA-Z0-9._$-]+$/', $name)) {
```

Comment:
Same: this regex really sucks, it doesn't allow array filters like `?foo[]=bar`, but this pattern is required by API Gateway on Swagger doc import